stories/Highlighter.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, select, text, withKnobs } from '@storybook/addon-knobs';
+import { StringHighlighter, HighlighterCoreAccuracy } from '@textkernel/oneui';
+
+storiesOf('Atoms|Highlighter', module)
```

Comment:
A very minor one but it doesn't look like an atom to me, the logic and the structure is pretty complex so I'd move it at least to Molecules 😄

Reply:
I think `Molecule ` is a component that made up of other components. The current component is a standalone component and that does not matter how is code big or complex.

Reply:
Actually it is a small component, just 46 lines :) Probably you meant `HighlighterCore`, but it is not React component.